lib/writer/object_model_writer.dart

```diff
@@ -137,6 +166,9 @@ class ObjectModelWriter {
       jsonModel.fields.forEach((field) {
         sb.write(' &&\n          ${field.name} == other.${field.name}');
       });
+      if (jsonModel.extend != null) {
```

Comment:
This assumes that the model being extended implements equals...

Reply:
Equals has a default implementation, what do you suggest we do here?